tests/test_normal.py

```diff
@@ -1 +1,14 @@
-# Last names: Q, R
\ No newline at end of file
+# Last names: Q, R
+from minecraft_gen import compute_normal_map, get_normal_light, get_normal_map
+import numpy as np
+
+def test_get_normal_light():
+    height_map = np.array([[0.0, 0.5], [0.8, 1.0]])
+    expected_result = np.array([ [0.65027821, 0.65267539], [0.60773718, 0.61224008]])
+
+    result = get_normal_light(height_map)
+    
+    assert result.shape == expected_result.shape
+    tolerance = 1e-6
+    assert np.allclose(result, expected_result, rtol=tolerance, atol=tolerance)
```

Comment:
It's better to use `np.testing.assert_allclose`, which provides custom error messages if the values don't match. However, even better is using the built-in support for numpy arrays in `pytest.approx`:

```suggestion
    assert result == pytest.approx(expected_result, rel=tolerance, abs=tolerance)
```

This provides even nicer custom error messages and doesn't require knowledge of the `np.testing` module.